Zend/tests/error_reporting02.phpt

```diff
@@ -9,7 +9,7 @@ function foo($arg) {
 }
 
 function bar() {
-	error_reporting(E_ALL|E_STRICT);
+	error_reporting(E_ALL|E_DEPRECATED);
```

Comment:
This test doesn't make sense (and hasn't since E_STRICT was included in E_ALL). I think what we need is to use something like `error_reporting(E_ALL & ~E_DEPRECATED)` above and then `error_reporting(E_ALL)` here, to preserve the spirit of the original test.